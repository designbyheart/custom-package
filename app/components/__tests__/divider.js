import { Text } from "react-native";
import React from "react";
import Divider from "../divider";

// Note: test renderer must be required after react-native.
import renderer from "react-test-renderer";

it("should render divider with left and right", () => {
  const divider = renderer
    .create(<Divider left={<Text>Test Text</Text>} />)
    .toJSON();
  expect(divider).toMatchSnapshot();
});
