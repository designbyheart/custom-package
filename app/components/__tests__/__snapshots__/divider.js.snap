// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render divider with left and right 1`] = `
<View
  style={
    Array [
      Object {
        "flex": 1,
      },
      Object {
        "backgroundColor": "#EBEBEA",
        "flexDirection": "row",
        "paddingHorizontal": 10,
        "paddingVertical": 5,
      },
      undefined,
    ]
  }
>
  <View
    style={
      Array [
        Object {
          "flex": 1,
        },
        Object {
          "alignItems": "flex-start",
        },
      ]
    }
  >
    <View>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
      >
        Test Text
      </Text>
    </View>
  </View>
  <View
    style={
      Array [
        Object {
          "flex": 1,
        },
        Object {
          "alignItems": "flex-end",
        },
      ]
    }
  >
    <View />
  </View>
</View>
`;
